vitasdk-update

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+
+if [ -z "$VITASDK" ]; then
+  echo '$VITASDK is not set'
+  exit 1
+fi
+
+. $DIR/include/install-vitasdk.sh
```

Comment:
Note that renamed headers and stubs will be duplicated when this script is executed.